vcd_cli/role.py

```diff
@@ -36,3 +38,30 @@ def list_roles(ctx):
         stdout(result, ctx)
     except Exception as e:
         stderr(e, ctx)
+
+
+@role.command('list_rights', short_help='list rights of a role')
+@click.pass_context
+@click.argument('role-name',
+                metavar='<role-name>',
+                required=True)
+@click.option('-o',
+              '--org',
+              'org_name',
+              required=False,
+              metavar='<org-name>',
+              help='name of the org')
+def list_rights(ctx, role_name, org_name=None):
+    try:
+        client = ctx.obj['client']
+        if (org_name is not None):
+            org_href = client.get_org_by_name(org_name).get('href')
+        else:
+            org_href = ctx.obj['profiles'].get('org_href')
+        org = Org(client, org_href, is_sysadmin(ctx))
+        role_record = org.get_role(role_name)
+        role = Role(client, href=role_record.get('href'))
+        rights = role.list_rights()
```

Comment:
As I pointed out in the pyvcloud review, I think it would be great if we can also list the id of the roles along with their names.

Reply:
None of the current list subcommands of other entities like org,vapp etc list ids.
We can retrieve ids using search command.

Reply:
agree, usually ids are not shown in `list`, just in the `info` sub command